upload_events.py

```diff
@@ -130,6 +130,16 @@ def upload_events(file_name: Path, dsn: str, project_id: int, project_slug: str,
 
                 event.pop('debug_meta', None)
 
+                # make sure the event does not go through minidump processing,
+                # otherwise we get weird errors in the pipeline we have to
+                # ignore
+                exceptions = get_path(event, "exception", "values", filter=True)
+                if exceptions:
+                    mechanism = get_path(exceptions, 0, "mechanism", "type"):
+
+                    if mechanism:
```

Comment:
Why not `if mechanism == "minidump":`? Are there other values we need to disable?

Reply:
yeah, applecrashreport